coalib/misc/Future.py

```diff
@@ -0,0 +1,83 @@
+from functools import partial
+
+
+# Start ignoring all
```

Comment:
As this is a copy and is not completely PEP8 conform ive deactivated all bears ^